website/content/index.markdown

```diff
@@ -166,6 +166,9 @@ files to keep your CLI description short and readable. You can then use
 Thor makes it easy to specify options and flags as metadata about a Thor
 command:
 
+ # Note  
+  option(and options)  are aliases for method_option (and methodo_options) respectively.
```

Comment:
There's a typo above. And I'd probably switch this around and say that `method_option` is an alias for `option` since the latter is much more commonly used.

Reply:
@dorner Thanks for the catch! fixed.